README.md

```diff
@@ -13,45 +13,12 @@ the [Nest framework](https://docs.nestjs.com/) for the NodeJS server.
 Configuration for the project is currently managed using environment variables.
 Docker handles the setting of key-value pairs in `docker-compose.yml`,
 through the `environment` block and by consuming a `.env` file.
-Docker will expect to find a `.env` file at the project's root directory.
+Docker will expect to find a `.env` file at the ${HOME}/secrets/ccm/.env directory.
 
 This repository includes in the `config` directory a `.env.sample` template file for `.env`,
 with the expected keys provided. Comments above each key describe what the value is used for,
 and in some cases, indicate that the key-value pair is optional.
 
-The `config.ts` module in the `ccm_web/server` directory validates this file,
-using fallbacks when available and throwing an error if required values are not present.
-The application will exit if any of the required configuration values are not set properly.
-
-#### Canvas
```

Comment:
I think much of this section is still relevant. Might want to break it out the parts that won't change into separate .MD files to make it easier to maintain separately like we did for MyLA in `docs`.